### ISSUE TITLE
Bug report. Platform not support 'x86_64-w64-mingw32'. Thank you.

### ISSUE THREAD

**Issue by ChengweiLou** [kahypar/KaHyPar.jl#9]
┌ Error: Error building `KaHyPar`: 
│ ERROR: LoadError: Your platform ("x86_64-w64-mingw32", parsed as "x86_64-w64-mingw32-gcc8-cxx11") is not supported by this package!
│ Stacktrace:
│  [1] error(::String) at .\error.jl:33
│  [2] top-level scope at C:\Users\2544035L\.julia\packages\KaHyPar\52iYf\deps\build.jl:47
│  [3] include(::String) at .\client.jl:439
│  [4] top-level scope at none:5
│ in expression starting at C:\Users\2544035L\.julia\packages\KaHyPar\52iYf\deps\build.jl:39
└ @ Pkg.Operations D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Pkg\src\Operations.jl:899

**Comment by SebastianSchlag** [comment]
I guess this pretty much boils down to us currently not supporting Windows / mingw right now. @jalving Whats your take on this?

**Comment by jalving** [comment]
@SebastianSchlag That's correct.  Has you ever successfully compiled a windows binary?  It looks like you tried building on Appveyor but it failed to find Boost.

**Comment by SebastianSchlag** [comment]
The Aappveyor builds once worked, but I haven't maintained them for quite a while, since at some point Appveyor started to take more than an hour to build the project and the CI jobs then timed out.

**Comment by jalving** [comment]
I'll take a stab at compiling KaHyPar on a windows VM when I get a chance.  If you have a MacOS build, we could also put that under https://github.com/jalving/KaHyParBuilder.  

Additionally, Julia's binary support looks better than it was 2 years ago.  They have pre-compiled packages for cmake and boost now which should speed up compilation.  It might be possible to get KaHyPar to auto-build and host binaries.

**Comment by ChengweiLou** [comment]
> I'll take a stab at compiling KaHyPar on a windows VM when I get a chance. If you have a MacOS build, we could also put that under https://github.com/jalving/KaHyParBuilder.
> 
> Additionally, Julia's binary support looks better than it was 2 years ago. They have pre-compiled packages for cmake and boost now which should speed up compilation. It might be possible to get KaHyPar to auto-build and host binaries.

Thank you.
I just move to Linux. Consider there may be other packages for my research that may not support windows so well in the future.
